src/software/ai/hl/stp/tactic/goalie_tactic.cpp

```diff
@@ -264,8 +264,8 @@ void GoalieTactic::calculateNextAction(ActionCoroutine::push_type &yield)
 
             // we want to restrict the block cone to the friendly crease, also potentially
             // scaled by a defense_area_deflation_parameter
-            Rectangle deflated_defense_area = field.friendlyDefenseArea();
-            deflated_defense_area.expand(-defense_area_deflation);
+            Rectangle deflated_defense_area =
+                *field.friendlyDefenseArea().expand(-defense_area_deflation);
```

Comment:
Should we not check that `expand` didn't return`std::nullopt` here?

Reply:
yeah that's a detail that was missing in the existing code. It really should've checked for expand returning false